README.md

```diff
@@ -65,6 +66,7 @@ Example response for /martha_v3:
     "name": "dd3c716a-852f-4d74-9073-9920e835ec8a/f3b148ac-1802-4acc-a0b9-610ea266fb61",
     "gsUri": "gs://my-bucket/dd3c716a-852f-4d74-9073-9920e835ec8a/f3b148ac-1802-4acc-a0b9-610ea266fb61",
```

Comment:
I don't understand how this standard was usable at all before the addition of `fileName`

If someone gives me a URI like `drs://test/some-uuid` and I dereference it, I get the GCS path... but no usable file paths. Am I supposed to `gsutil ls` in my code to get the filenames? Seems like a big omission

This is a rounadabout way of saying, good thing you implemented this, it seems essential

Reply:
JDR was the first server that was a) using this optional field and b) that we noticed. The DOS/DRS servers currently being used in production return GCS paths that end in usable file names so no one has complained yet.

Reply:
Ohhhh whaaaaaat? How is it possible to write code that consumes these URIs without knowing whether it points to a file or not? Clients will have to write sensor code to parse the string and guess whether it's a file or not

Reply:
Yeah, we can discuss within WA and will have to extend discussions to the GATK, but this is not new, just not common in bioinformatics IMHO.

There's been a constant battle of "please use the file extension to define the file type" for decades. Still historically [MIME types](https://en.wikipedia.org/wiki/Media_type), [MacOS resource forks](https://en.wikipedia.org/wiki/Resource_fork), and now DOS/DRS have had separate concepts of file names vs. file types.